test/helpers/env.js

```diff
@@ -203,6 +214,7 @@ env.IOS82 = env.DEVICE.match(/ios82/i);
 env.IOS83 = env.DEVICE.match(/ios83/i);
 env.IOS84 = env.DEVICE.match(/ios84/i);
 env.IOS9 = env.DEVICE.match(/ios9/i);
+env.IOS91 = env.DEVICE.match(/ios91/i);
```

Comment:
Are there any operations in the codebase that only affect iOS 9? If not, this is ok. If there are, env.iOS9 will be turned on when it is iOS 9.1 also (i.e., the regular expression for iOS 9 here will also match for iOS 9.1).